bitnami/influxdb/Chart.yaml

```diff
@@ -15,4 +15,4 @@ maintainers:
 name: influxdb
 sources:
   - https://github.com/bitnami/bitnami-docker-influxdb
-version: 0.4.7
+version: 0.5.7
```

Comment:
This shouldn't be `0.5.7`, but `0.5.0` instead

Reply:
Fixed in latest commit